generator/src/main/java/com/scottlogic/deg/generator/violations/ViolatedProfile.java

```diff
@@ -7,8 +7,19 @@
 import java.util.Collection;
 
 public class ViolatedProfile extends Profile {
+    /**
+     * Original (un-violated) form of the rule that has been violated in this profile.
+     */
     public final Rule violatedRule;
```

Comment:
How about rename this to ruleToViolate and cut the comment.

Reply:
@pdaulbyscottlogic That suggests to me that something is going to be done to this rule though?

Reply:
It's already been violated, this is just the unviolated form of the one that's been violated

Reply:
would ruleThatWasViolated be too awful

Reply:
Yes